app/uk/gov/hmrc/taxenrolmentassignmentfrontend/testOnly/connectors/IdentityProviderAccountContextConnectorTestOnly.scala

```diff
@@ -87,4 +87,47 @@ class IdentityProviderAccountContextConnectorTestOnly @Inject() (
       }
   }
 
+  def deleteIndividual(caUserId: String)(implicit hc: HeaderCarrier): TEAFResult[Unit] = {
+    val url =
+      s"${appConfigTestOnly.identityProviderAccountContextBaseUrl}/identity-provider-account-context/test-only/test/accounts/$caUserId"
+
+    EitherT(
+      httpClient.DELETE[Either[UpstreamErrorResponse, HttpResponse]](
+        url
+      )
+    ).transform {
+      case Right(response) if response.status == OK => Right(())
+      case Right(response) =>
+        val ex = new RuntimeException(s"Unexpected ${response.status} status")
+        logger.error(ex.getMessage, ex)
+        Left(UpstreamUnexpected2XX(response.body, response.status))
+      case Left(upstreamError) =>
+        logger.error(upstreamError.message)
+        Left(UpstreamError(upstreamError))
+
+    }
+  }
+
+  def getEacdIds(nino: String)(implicit hc: HeaderCarrier): TEAFResult[List[String]] = {
+    val url =
+      s"${appConfigTestOnly.identityProviderAccountContextBaseUrl}/identity-provider-account-context/contexts/individual/$nino"
+
+    EitherT(
+      httpClient.GET[Either[UpstreamErrorResponse, HttpResponse]](
+        url
+      )
+    ).transform {
+      case Right(response) =>
+        val listOfCreds: List[Creds] =
+          (response.json \ "credentials").as[List[Creds]]
+
+        Right(listOfCreds.map { x =>
+          x.caUserId
```

Comment:
I'm sure there's a better way of doing this, but my Friday afternoon brain cannot figure out what it is :(

Reply:
Well x => x  can be changed to just _ for a start